### DISCUSSION TITLE
8.2.1.1 unable to download stream

### DISCUSSION THREAD

**Discussion by githubszero333** [aclap-dev/video-downloadhelper#240] [Help and Q&A]
### Your question

==========================
url:  https://live.fc2.com/
or      https://live.fc2.com/70395754/
==========================
Exit code: 1
Error: Exit code: 1

    at rs.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624
====================================
how to fix it?

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: zh-TW
license: accepted
key (only 16 first characters): b3996dcd6c334c13
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\MYPC",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3235, Firefox 123.0 64-bit, licensed VDH 8.2.2.0a1 beta, CoApp 2.0.19.  Note that I do not work for VDH.  I am just another user, like you.

I visited the second link you provided & it looked like this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8dbf87f2-f1c8-4287-adce-dd5d96be7e94)

Note that I had not clicked the play button in the player.  I launched the download of the first variant you can see on that VDH menu.  That got me this:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/35a733ea-263f-4145-be6e-8604d17755b6)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/62bd8923-8151-4825-9ee0-7b9b7aaa560c)

After a few minutes, I clicked the X within the black circle that you can see in those last 2 images.  Here are the results:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/d2875824-f4ad-4a8d-892d-a90b25f1e332)

And here it is playing in VLC:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/4c3f8885-e001-4554-8c51-b42697c1ad01)

I didn't sit & watch it, just sampled it at intervals.  As you can see, it played fine.  You'll have to take my word for it that the audio was equally fine, although I didn't understand a word of it since it was a language I don't speak.

Conclusion: Get the latest software.  I know you're using Chrome.  But there are betas for Chrome.  I'm not sure if the Chrome beta is as up to date as the Firefox beta.  If not, I'm sure @paulrouget will put one up soon.

**Reply by ebguy06** [reply]
**Yes, why does the timer keep going up forever, 300+ hours for every hour?**

I havent been able to download streaming for over two weeks now, everything was fine, I revert to older versions & coapp but wont work anymore.

File timer goes up forever, download speed drops from 10mbs to kbs within few minutes _[fixed with 8.2.2.0a1 (beta)]_, downloader stops after two minutes _[fixed 8.2.2.0a1 (beta)]_; 

**I no longer see the details of the downloading, ex: resolution or type, same as your image#2.**


`
version: 8.2.2.0a1
channel: beta
build date: 2024-03-01
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\oneof",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
`

**Comment by ebguy06** [comment]
I'm so tired of this... **I bet your issues started on/after 2/14/2024 ?**
Every version since has been worse than the last, even betas.
For some insane reason after 15hours I cannot get anything to return to prior to 2/14, both older versions including coapp vers.
One thing I didnt try was going to an older firefox version from that date range.


> ### Your question
> # ==========================
> url: https://live.fc2.com/
> or https://live.fc2.com/70395754/
> 
> Exit code: 1 Error: Exit code: 1
> 
> ```
> at rs.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
> at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624
> ```
> 
> ==================================== how to fix it?
> ### Addon & CoApp version and details
> 
> version: 8.2.1.1 channel: stable build date: 2024-02-19 build options: linuxlic: false, noyt: true, browser: chrome lang: zh-TW license: accepted key (only 16 first characters): b3996dcd6c334c13 platform: x86-64 win UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 coapp: yes. Info: { "id": "net.downloadhelper.coapp", "name": "VdhCoApp", "version": "2.0.19", "binary": "C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe", "displayName": "VdhCoApp", "description": "Video DownloadHelper companion app", "target": { "os": "windows", "arch": "x86_64", "node": "18" }, "home": "C:\Users\MYPC", "converterBinary": "C:\Program Files\DownloadHelper CoApp\ffmpeg.exe", "converterBase": "ffmpeg", "converterBaseVersion": "c30f360955" }

**Reply by GrampaWildWilly** [reply]
I don't understand why you're having a problem.  We're both running Windows.  You are (or were) running the same VDH beta that I am, the one that Paul put up last night: 8.2.2.0a1 beta.  You're running the same CoApp I am: 2.0.19.  You're running a different browser, although I see above that you tried Firefox 123.0, which is what I have.  It recorded for me without a problem.  That's no help to you but it is surprising you're having problems.

The constantly increasing time remaining is typical of livestreams while they are in progress.  The livestream hasn't ended so there is no way to know how long it will be.  The ever increasing time remaining is a result of VDH discovering progressively more of the livestream that has already occurred.  I believe -- I haven't tested this hypothesis, plus I don't work for VDH so I don't have access to the code -- I believe VDH starts the recording of a livestream at the beginning, not at the moment you tell VDH to start recording it.  So the early part of what VDH downloads is whatever part of the livestream has been archived on the site, the part that occurred before now.  There is a certain period during which VDH is catching up.  That would normally download pretty fast, assuming the server isn't throttling their service too badly.  Once VDH catches up, the download speed would drop because it is finally recording the current feed.  Video feeds don't usually take much bandwidth.  Typically, a livestream takes only a few hundred thousand bytes per second of bandwidth.  So the speed dropping after a time might not be a concern.  But like I say, this is just based on my observations over time, not based on how the code is written because I've never seen the code.  I probably wouldn't understand it anyway.  I'm just another user, like you.

I don't know why there would be no report of the video resolution of the feed.  @paulrouget would have to answer that.  Unlike me, he actually does work for VDH.  He's been writing the code.

The only other idea I have is that maybe you're hitting a problem with either your antivirus or your firewall or both.  There have been a number of discussions in here on that topic.  There is also a discussion of that in the Wiki.  You should get in the habit of clicking the Wiki link at the top of any page of this repository.  Any time you have a question, you should check the Wiki to see if a topic is covered.  Both antivirus & backlevel releases are discussed in the Wiki.
